Formula/coredns.rb

```diff
@@ -0,0 +1,89 @@
+
+class Coredns < Formula
+  desc "DNS server that chains plugins"
+  homepage "https://coredns.io"
+  url "https://github.com/coredns/coredns/archive/v1.0.3.tar.gz"
+  sha256 "dedc42f1a4aac56e397a3fff19955327f21460782aef870658fa1353134ab2f7"
+  head "https://github.com/coredns/coredns.git"
+
+  def default_coredns_config; <<~EOS
```

Comment:
Since you're only using this once inline it with the `(buildpath/"Corefile.example").write` below.